.github/workflows/ci.yml

```diff
@@ -321,34 +327,45 @@ jobs:
 
       # run tests
       - name: Test repo
-        run: ${{ env.GOTESTCMD }} -gcflags=all=-d=checkptr -tags admin ./...
+        run: ${{ env.GOTESTSUM_CMD }} -gcflags=all=-d=checkptr -tags admin -timeout=20m ./...
```

Comment:
why does only this one need a timeout?

Reply:
the default timeout of 10 minutes was cutting it close in some cases, i think